### ISSUE TITLE
Missing gradle-wrapper.jar

### ISSUE THREAD

**Issue by Emkas** [manticore-projects/H2MigrationTool#10]
I was trying to use online version of the tool, but it was unable to migrate any table. So I wanted to build it from source and debug. But there is no place to start since `gradle-wrapper.jar` is missing from `gradle/wrapper` directory.

This is probably because `.gitignore` is ignoring all `*.jar`, but this one jar is needed for `gradlew` to work.

**Comment by manticore-projects** [comment]
Greetings!

Thank you for reporting, I have pushed the Wrapper to the repository.
Also, you can download Fat Jars from: http://manticore-projects.com/download/H2MigrationTool-1.4/H2MigrationTool-1.5-SNAPSHOT-all.jar

Last but not least please elaborate on "but it was unable to migrate any table". What exactly went wrong?

Cheers!

**Comment by manticore-projects** [comment]
I have just tried the Online Demo by myself with a rather large DB (105MB) and it worked as expected.
So I am keen to see it from your point of view in order to improve the process.

**Comment by Emkas** [comment]
> Thank you for reporting, I have pushed the Wrapper to the repository.

Great to hear!
Now I can almost get a build running:

```
> Task :spotbugsMain FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':spotbugsMain'.
> A failure occurred while executing com.github.spotbugs.snom.internal.SpotBugsRunnerForHybrid$SpotBugsExecutor
   > Verification failed: SpotBugs ended with exit code 1. See the report at: file:///home/emkas/inne_projekty/H2MigrationTool/build/reports/spotbugs/main/spotbugs.html
```

> Also, you can download Fat Jars from: http://manticore-projects.com/download/H2MigrationTool-1.4/H2MigrationTool-1.5-SNAPSHOT-all.jar

Thank you for this, I wasn't looking eager enough.

> Last but not least please elaborate on "but it was unable to migrate any table". What exactly went wrong?

Well, there was no error, but the database was empty. I run it over several times. It is very small database (~30kb) for the spring boot app with 3 tables and flyaway.

**Comment by Emkas** [comment]
OK, those are the same issues from Spotbugs that are on your action: https://github.com/manticore-projects/H2MigrationTool/actions/runs/6556123622/job/17805622513.

I've changed 9 places by adding `@SuppressFBWarnings` and I went great from there. I run GUI in eclipse and migrate the database without any issues.

I can say e.g. that returned from online tool files was named like my original file, and in GUI files i sufixed with `<DEST_VERSION>.db`.
